steps/identity/appellant-in-uk/template.html

```diff
@@ -4,7 +4,7 @@
 {% from "details/macro.njk" import govukDetails %}
 
 {% block pageTitle %}{{ content.titleHead }} - {{ commonContent[htmlLang].windowTitleSuffix }}{% endblock %}
-{% set title = content.title %}
+{% set title = title %}
```

Comment:
is this needed anymore?

Reply:
suppose not! will remove (from other instances too)